cumulusci/tasks/bulkdata/tests/test_snowfakery.py

```diff
@@ -74,6 +75,25 @@ def temporary_file_path(filename):
         yield path
 
 
+@pytest.fixture()
+def ensure_accounts(create_task, run_code_without_recording, sf):
+    """Delete all accounts and create a certain number of new ones"""
```

Comment:
So if you accidentally run this against the wrong org, it's destructive. Should we have some check to confirm that the tests can only run against scratch orgs?

Reply:
@prescod this comment was buried by github and I think it's important; did you see it?

Reply:
That code that translates orgnames to orgconfigs has that check already:

* https://github.com/SFDO-Tooling/CumulusCI/blob/246e75f49e89b3d269a9613d68e932c4363cf9fb/cumulusci/tests/pytest_plugins/pytest_sf_orgconnect.py#L50